### DISCUSSION TITLE
Looking for feedback: Rename `stimpack` to `packs-rails`

### DISCUSSION THREAD

**Discussion by alexevanczuk** [rubyatscale/packs-rails#49] [General]
Hi all!

With the recent move to extract `packs` out from `stimpack` (https://github.com/rubyatscale/packs), we are looking to rebrand some of the rubyatscale ecosystem around the idea of ruby "packs."

With this in mind, we thought that `stimpack` is really `packs-rails`. It could technically even be split out into `packs-rails`, `packs-rspec`, and `packs-factorybot`, but we'd likely want to keep everything in `packs-rails` for now to keep things simple.

We're looking to see if the community has any feedback/concerns/objections they'd like to share before we move ahead with this!

This would mean:
- Referencing `packs-rails` in your `Gemfile`
- Using the constant `Packs::Rails` instead of `Stimpack`
- If the location of your packs is something other than `packs/*/package.yml`, creating a `packs.yml` file, see https://github.com/rubyatscale/packs for more info. This replaces `Stimpack.config.root = '...'`.

**Comment by alexevanczuk** [comment]
PR: https://github.com/rubyatscale/stimpack/pull/51/files

**Comment by silasb** [comment]
I think this makes sense.  While you guys do a good job making adoption easy, it might be easier if we had separate integrations that we can do to migrate into `pack-rails`.  During a recent working grouping meeting, we'd love to have a solution for `factorybot` within `packwerk` packages.  Having `packs-factorybot` could make that an easier migration path.

**Reply by alexevanczuk** [reply]
@silasb `packs-rails` does actually support factorybot out of the box – it just hasn't yet been factored out into a separate gem. I might be misunderstanding your comment though -- are you saying it would be nice if there were a factorybot integration, or it would be easier if it were a separate gem? If the latter, could you elaborate?
